examples/inet-frags.bpf.c

```diff
@@ -0,0 +1,39 @@
+#include <vmlinux.h>
+#include <bpf/bpf_core_read.h>
+#include <bpf/bpf_tracing.h>
+#include "maps.bpf.h"
+
+struct inet_frags_key_t {
+    u32 ifindex;
+    u8 ip_version;
+};
+
+struct {
+    __uint(type, BPF_MAP_TYPE_HASH);
```

Comment:
Let's use `BPF_MAP_TYPE_LRU_HASH` with a smaller `max_entries` (64 interfaces is a reasonable assumption).